docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -142,6 +183,197 @@ data:
     [“172.16.0.1”]
 ```
 
+## Debugging DNS resolution
+
+### Create a simple Pod to use as a test environment
+
+Create a file named busybox.yaml with the following contents:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox
+  namespace: default
+spec:
+  containers:
+  - image: busybox
+    command:
+      - sleep
+      - "3600"
+    imagePullPolicy: IfNotPresent
+    name: busybox
+  restartPolicy: Always
+```
+
+Then create a pod using this file and verify its status:
+
+```
+kubectl create -f busybox.yaml
```

Comment:
Adding a `$` before the command (and following commands) might make it more clear.